pkg/cmd/release/list/http.go

```diff
@@ -76,3 +88,20 @@ loop:
 
 	return releases, nil
 }
+
+func (r *Release) ExportData(fields []string) map[string]interface{} {
```

Comment:
This exact function has been copy/pasted a couple times in the code base, it would be nice to have a single place for it. I don't think this needs to be addressed in this PR but just a note for the future.